pkg/ovs/ovs-vsctl.go

```diff
@@ -206,7 +206,23 @@ func CleanLostInterface() {
 				return
 			}
 			klog.Infof("delete lost port %s", name)
-			output, err := Exec("--if-exists", "--with-iface", "del-port", "br-int", name)
+
+			ifUUID, err := ovsFind("interface", "_uuid", fmt.Sprintf("name=%s", name))
+			if err != nil {
+				klog.Errorf("failed to find related interface %v uuid %v", name, err)
+				return
+			}
+			portUUID, err := ovsFind("port", "_uuid", fmt.Sprintf("interfaces{>=}%s", ifUUID[0]))
+			if err != nil {
+				klog.Errorf("failed to find related port %v uuid %v", ifUUID[0], err)
+				return
+			}
+			briName, err := ovsFind("bridge", "name", fmt.Sprintf("ports{>=}%s", portUUID[0]))
+			if err != nil {
+				klog.Errorf("failed to find related bridge %v uuid %v", portUUID[0], err)
+				return
+			}
+			output, err := Exec("--if-exists", "--with-iface", "del-port", briName[0], name)
```

Comment:
We can use `ovs-vsctl del-port <PORT>`, so there is no need to get the bridge name.

Reply:
fixed